<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesAction</class>
 <widget class="QDialog" name="NotesAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Actions en masse</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string notr="true">Export</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="export_csv_button">
            <property name="text">
             <string notr="true">Export CSV</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string notr="true">Consommations</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1" colspan="2">
           <widget class="UniqueConsumptionList" name="product_list">
            <column>
             <property name="text">
              <string notr="true">Produits</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="take_products_btn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string notr="true">Valider</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QPushButton" name="refill_button">
            <property name="text">
             <string notr="true">Recharger</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="price_list">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QPushButton" name="take_button">
            <property name="text">
             <string notr="true">Prendre</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" rowspan="2" colspan="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string notr="true">Notes sélectionnées</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="FilteredNotesListWidget" name="note_list" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QTreeWidget" name="selected_notes_report">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string notr="true">Note</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">Quantité</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="notes_management_button">
     <property name="text">
      <string notr="true">Gerer les notes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UniqueConsumptionList</class>
   <extends>QTreeWidget</extends>
   <header>gui/group_actions_window</header>
   <slots>
    <slot>on_selection()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>FilteredNotesListWidget</class>
   <extends>QWidget</extends>
   <header>gui/filtered_notes_list_widget</header>
   <container>1</container>
   <slots>
    <signal>itemSelectionChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>export_csv_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>export_csv_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>take_products_btn</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>take_products_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>product_list</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>product_list</receiver>
   <slot>on_selection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refill_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>refill_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>take_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>take_action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>note_list</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>NotesAction</receiver>
   <slot>selection_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selected_notes_report</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>NotesAction</receiver>
   <slot>check_edit(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notes_management_button</sender>
   <signal>clicked()</signal>
   <receiver>NotesAction</receiver>
   <slot>notes_management_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>786</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>del_action()</slot>
  <slot>hide_action()</slot>
  <slot>show_action()</slot>
  <slot>filter_combobox_change()</slot>
  <slot>filter_input_changed()</slot>
  <slot>export_xml_action()</slot>
  <slot>export_csv_action()</slot>
  <slot>take_products_action()</slot>
  <slot>refill_action()</slot>
  <slot>take_action()</slot>
  <slot>search_input_changed()</slot>
  <slot>selection_changed()</slot>
  <slot>check_edit(QTreeWidgetItem*,int)</slot>
  <slot>notes_management_fnc()</slot>
 </slots>
</ui>
